akaunting/akaunting_stackscript.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# <UDF name="company_name" Label="Company Name" default="My Company" />
+# <UDF name="company_email" Label="Company Email" default="my@company.com" />
+# <UDF name="admin_email" Label="Admin Email" default="my@company.com" />
+# <UDF name="admin_password" Label="Admin Password" default="123654" />
+
+# <UDF name="db_name" Label="MySQL Database Name" default="akaunting" />
+# <UDF name="db_password" Label="MySQL root Password" default="123654" />
+# <UDF name="dbuser" Label="MySQL Username" default="akaunting" />
+# <UDF name="dbuser_password" Label="MySQL User Password" default="123654" />
```

Comment:
These UDF's are presented as additional options when deploying the stackscript. Is this intentional? If these are required fields for the application, I would recommend removing the "default=12345" and using "example=s3c0rePa55worD"